src/main/java/ne/fnfal113/relicsofcthonia/RelicsOfCthonia.java

```diff
@@ -27,12 +27,13 @@ public final class RelicsOfCthonia extends JavaPlugin implements SlimefunAddon {
     public void onEnable() {
         setInstance(this);
 
-        new Metrics(this, 15420);
+        //new Metrics(this, 15420);
```

Comment:
re-enable this please, so I can track builds in bstats

Reply:
Okay, I will re-enable this back.
Because on Slimefun Official Discord, they say you need to turn off the bstats to make sure this unofficial jar won't affect their addons bstats metric, this is why all of addons translate bstats is default false.

Reply:
A very long time ago
https://discord.com/channels/565557184348422174/565794027518296074/769178681344000000